test/es-module/test-typescript.mjs

```diff
@@ -324,3 +324,32 @@ test('execute a JavaScript file importing a cjs TypeScript file', async () => {
   match(result.stdout, /Hello, TypeScript!/);
   strictEqual(result.code, 0);
 });
+
+test('execute a TypeScript file with Union Types', async () => {
+  const result = await spawnPromisified(process.execPath, [
+    '--experimental-strip-types',
+    '--no-warnings',
+    fixtures.path('typescript/ts/test-union-types.ts'),
+  ]);
+
+  strictEqual(result.stderr, '');
+  match(result.stdout, /Hello, TypeScript!/);
+  match(result.stdout, /admin/);
+  match(result.stdout, /Testing Partial Type/);
```

Comment:
```suggestion
  strictEqual(result.stdout, 'Hello, TypeScript!\nadminTesting Partial Type\n')
```

Can't you check the entire stdout?

Reply:
Thanks!!

Reply:
This test didn't work

Reply:
Check what the output should be, and match that.

Reply:
Updated! 🙏